lib/virt_autotest/utils.pm

```diff
@@ -156,6 +156,7 @@ sub check_host_health {
         return 'pass';
     }
     record_info("Unhealthy host", $failures, result => 'fail');
+    upload_system_logs;
```

Comment:
I can understand that logs are uploaded when there are failures, but for our virtualization tests, it is a different case. The host is checked before each test module, once 'core dump'/'call trace' are found in journal files on the machine, they will be reported in each check in all test modules. fg. https://openqa.suse.de/tests/10188966
In this case, we have 18 times check, it would be meaningless and unrealistic to upload similiar logs 18 times. 

Considering our case, could you come up with a better solution to this problem?

Reply:
I don't see a problem with having too many logs. Logfiles uploaded by different test modules are prefixed by the test module name so you can immediately see which logfile was uploaded at which point in the job. Then you can quickly review them by looking at the first one and the last one to see whether it's all the same error or different test modules produced additional errors during the job. And then you can check the logfiles in between as needed.

Reply:
If you insist on collecting logs only for new errors, you can use the `journalctl --cursor-file=...` option to only read journal entries that had not been checked before. However, that is outside the scope of this PR.

Reply:
As I said in [this comment](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16099#discussion_r1048411471), you can also use it in your way and does not break any others.

Reply:
I think `journalctl --cursor-file=` may be a better solution for both needs. fg.
```
sub check_failures_in_journal {
...
my $cmd = "journalctl --cursor-file=FILE | grep '$warn'";
```
then test only fails once with the first spot of failures. 'FILE' can be removed in the test where you'd like have an entire search.

Reply:
Using cursor is also option. I also agree if this can be done well.